src/notation/tests/mocks/msczreadermock.h

```diff
@@ -24,13 +24,13 @@
 
 #include <gmock/gmock.h>
 
-#include "notation/imsczmetareader.h"
+#include "project/imscmetareader.h"
 
 namespace mu::notation {
-class MsczReaderMock : public IMsczMetaReader
+class MsczReaderMock : public project::IMscMetaReader
```

Comment:
Msc ~~z~~ ReaderMock

Reply:
This needs to be fixed when moving tests.